server/planning/events/events.py

```diff
@@ -128,6 +138,20 @@ def is_event_updated(new_item: Event, old_item: Event) -> bool:
     return False
 
 
+def get_user_updated_keys(id: str) -> set[str]:
```

Comment:
Same here with using `set` for type, would need to use `Set` from `typing` module to support Python v3.8